semih-week2.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9];
+const isOdd = x => x % 2 !== 0;
+const double = x => x * 2;
+const doubleOdd = numbers.filter(isOdd).map(double);
+console.log(doubleOdd);
+
+// /////////////////////////////////
+
+const monday = [{
+    name: 'Write a summary HTML/CSS',
+    duration: 180,
+  },
+  {
+    name: 'Some web development',
+    duration: 120,
+  },
+  {
+    name: 'Fix homework for class10',
+    duration: 20,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+];
+
+const tuesday = [{
+    name: 'Keep writing summary',
+    duration: 240,
+  },
+  {
+    name: 'Some more web development',
+    duration: 180,
+  },
+  {
+    name: 'Staring out the window',
+    duration: 10,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+  {
+    name: 'Look at application assignments new students',
+    duration: 40,
+  },
+];
+
+const minToHour = x => x.duration / 60; // map
+const moreThanTwoHours = x => x > 2; // filter
+const perHourRate = x => x * 20; // map
+const sum = (x, y) => x + y; // reduce
+
+const mondayHours = monday.map(minToHour);
+const tuesdayHours = tuesday.map(minToHour);
+
+const mondayHoursFiltered = mondayHours.filter(moreThanTwoHours);
+const tuesdayHoursFiltered = tuesdayHours.filter(moreThanTwoHours);
+
+const mondayPerHourRates = mondayHoursFiltered.map(perHourRate);
+const tuesdayPerHourRates = tuesdayHoursFiltered.map(perHourRate);
+
+const mondayRateSum = mondayPerHourRates.reduce(sum, 0);
+const tuesdayRateSum = tuesdayPerHourRates.reduce(sum, 0);
```

Comment:
You can already notice that you are repeating the same stuff for monday and tuesday. This is always a good time to look at your code again and see if you can refactor it. If you use the `concat()` function I mentioned earlier you don't need to do the same thing for the two days.